### ISSUE TITLE
Enteroendocrine cell dysgenesis

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11198]
**Preferred term label:**
Enteroendocrine cell dysgenesis

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://uts.nlm.nih.gov/uts/umls/concept/C1969074

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://omim.org/entry/610370

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
What they actually observed in https://pubmed.ncbi.nlm.nih.gov/16855267/ was "Paucity of small intestinal enteroendocrive cells", and the inference was "dysgenesis". I think the former label is a bit clearer. Thoughts?

**Comment by MickeySegal** [comment]
Agree.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001346
- label: Enteric anendocrinosis
- parents: [Abnormal intestine morphology (HP:0002242)](https://hpo.jax.org/app/browse/term/HP:0002242)
- pmid: PMID:39346060|PMID:16855267
- def: Near absence of small intestinal and colonic endocrine cells. This feature can be ascertained histopathologically and may also be manifested as a lack of staining for enteroendocrive markers such as synaptophysin.
- comment: Enteroendocrine cells (EECs) are gut epithelial hormone-secreting cells, influenced by diet and the microbiome. EECs regulate local gastrointestinal functions and systemic signalling, including communication with the brain, while being linked to various diseases.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11198
- synonyms:
